resolwe_bio/tests/processes/test_reads_filtering.py

```diff
@@ -133,16 +133,14 @@ def test_trimmomatic_single(self):
         with self.preparation_stage():
             reads = self.prepare_reads()
 
-        inputs = {'reads': reads.pk,
-                  'trailing': 3,
-                  'crop': 5}
+        inputs = {'reads': reads.pk}
         filtered_reads = self.run_processor('trimmomatic-single', inputs)
 
         self.assertFiles(filtered_reads, 'fastq', ['filtered_reads_trimmomatic_single.fastq.gz'], compression='gzip')
         del filtered_reads.output['fastqc_url'][0]['total_size']  # Non-deterministic output.
         self.assertFields(filtered_reads, "fastqc_url", [{'file': 'fastqc/reads_fastqc/fastqc_report.html',
                                                           'refs': ['fastqc/reads_fastqc'],
-                                                          'size': 206718}])
+                                                          'size': 323419}])
```

Comment:
Size is not checked anymore [https://github.com/genialis/resolwe/blob/master/resolwe/test/testcases/process.py#L503](https://github.com/genialis/resolwe/blob/master/resolwe/test/testcases/process.py#L503).

Reply:
@JenkoB Please remove all occurrences of the size attribute in this tests.